cmd/main.go

```diff
@@ -92,7 +92,9 @@ func main() {
 	msgCache := messages.NewRedisCache(redisClient)
 
 	svc := export.New(cfg, msgCache, logger)
-	svc.Start(svcName)
+	if err := svc.Start(svcName); err != nil {
+		log.Fatalf(fmt.Sprintf("Failed to start service %s", err))
```

Comment:
Use our logger here. Also, instantiate the logger at the begin of the main to use it for any error.

Reply:
and I do, if you are refering to loadConfigs then I cannot instatniate logger when not knowing what log level and loadConfigs is loading from config file where all the settings for the app are stored